<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serverDialog</class>
 <widget class="QDialog" name="serverDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>178</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_connect">
      <property name="text">
       <string>Connect</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Server:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nick name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Alternative nick 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Alternative nick 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>User name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Real name:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_server">
        <property name="text">
         <string>irc.rizon.net</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_nick">
        <property name="text">
         <string>blaaaah</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_nick2">
        <property name="text">
         <string>wqerer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_nick3">
        <property name="text">
         <string>dfdfdff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_username">
        <property name="text">
         <string>cvcvcv</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_fullname">
        <property name="text">
         <string>hjhjhjhjhj</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
